modules/default/weather/weather.js

```diff
@@ -11,8 +11,10 @@ Module.register("weather", {
 	defaults: {
 		weatherProvider: "openweathermap",
 		roundTemp: false,
-		type: "current", //current, forecast
+		type: "current",
```

Comment:
could you add all new types to the comments you deleted? //so people can see what the possible values are?

Reply:
Whoops, I thought that was me, yeah I'll put that back. I've been adding them to the documentation as well.